src/foam/nanos/om/services.jrl

```diff
@@ -0,0 +1,7 @@
+p({"class":"foam.nanos.boot.NSpec", "name":"omLogger",                         "lazy":true,  "serve":false, "serviceClass":"foam.nanos.om.DAOOMLogger"})
+p({"class":"foam.nanos.boot.NSpec", "name":"omInfoDAO",                        "lazy":true,  "serve":true,  "serviceScript":"return new foam.dao.ProxyDAO.Builder(x).setDelegate(new foam.dao.MDAO(foam.nanos.om.OMInfo.getOwnClassInfo())).build();","client":"{\"of\":\"foam.nanos.om.OMInfo\"}" })
```

Comment:
Same comments as the other PR. Why are there plain ProxyDAO decorators on all of these? We should be wrapping these with `AuthenticationDAO`s.

Reply:
These were just copied over from the main ‘services’.  Want to switch to EasyDAO, but it has updates in another pending PR.

Reply:
removed.